### PR TITLE
Details the distributed.pipeline.sync.Pipe example

### PR DESCRIPTION
**Pull request by wangkuiyi**
1. Removed the leading `>>>` in each example code line so to unify the style with other examples in this package.
2. Explain the example is using one Python process to operate two GPUs.

### CODE REVIEW

File: torch/distributed/pipeline/sync/pipe.py
Comment (mrshenli) [inline torch/distributed/pipeline/sync/pipe.py:272]: do we need an additional indent to make it render as code?

Comment (wangkuiyi) [reply torch/distributed/pipeline/sync/pipe.py]: Add the indent. Thanks for the reminder!

**Review by mrshenli — APPROVED** [review]
Let's wait for doc build CI and verify docstrings are rendered correctly before landing.

**Comment by awgu** [comment]
It looks like the leading `>>>` might be needed?

Screen Shot 2022-08-09 at 10 46 00 PM

**Comment by wangkuiyi** [comment]
Yes, it seems that `>>>` are required. Let me abandon this PR.

**Pull request closed without merging**
